torch/distributed/algorithms/join.py

```diff
@@ -88,7 +88,7 @@ def _join_process_group(self) -> Any:
 
```

Comment:
I couldn't comment on line 81, so adding comments here. Any reason `_join_process_group` 's return type is Any? Is it because ProcessGroup is not a public type?

Reply:
Yes, that is the reason. I think for now, we have to type all process groups as `Any`.